cqlprotodoc/template-notice.txt

```diff
@@ -0,0 +1,5 @@
+Apache Cassandra
+Copyright 2009-2022 The Apache Software Foundation
```

Comment:
@tengu-alt not sure about the year ... @michaelsembwever thoughts? Do we even need the year at all?

Reply:
Do we even need the `template-notice.txt` file anymore? It was present in `cqlprotodoc` because the output of conversion of the Apache2-licensed text was published standalone on a separate website.

The Native Protocols page on apache.org already has ```© 2009-2024 The Apache Software Foundation under the terms of the Apache License 2.0. Apache, the Apache feather logo, Apache Cassandra, Cassandra, and the Cassandra logo, are either registered trademarks or trademarks of The Apache Software Foundation.``` in the footer, so I think we could remove the notice altogether.

Reply:
Agree, done. Also, i have made some refactoring in `main.go`